spec/models/case/sar/offender_spec.rb

```diff
@@ -218,15 +218,15 @@
       end
 
       it 'validates third party names when recipient is third party' do
-        kase = build :offender_sar_case, third_party: false, third_party_name: '', 
+        kase = build :offender_sar_case, third_party: false, third_party_name: '',
                       third_party_company_name: '', recipient: 'third_party_recipient'
         expect(kase).not_to be_valid
         expect(kase.errors[:third_party_name]).to eq ["can't be blank if company name not given"]
         expect(kase.errors[:third_party_company_name]).to eq ["can't be blank if representative name not given"]
       end
 
       it 'does not validate third_party names when ecipient is not third party too' do
```

Comment:
Small thing, also pre-existing this PR but should we change the spelling of `ecipient` here? Could make a list of this sorts of things and combine them in a later PR in one fell swoop.